pkg/cache/cluster.go

```diff
@@ -707,11 +707,7 @@ func (c *clusterCache) watchEvents(ctx context.Context, api kube.APIResourceInfo
 					err = runSynced(&c.lock, func() error {
 						openAPISchema, gvkParser, err := c.kubectl.LoadOpenAPISchema(c.config)
 						if err != nil {
-							e, ok := err.(*kube.CreateGVKParserError)
-							if !ok {
-								return err
-							}
-							c.log.Info("warning: failed to create gvk parser", "error", e.Error())
+							return err
```

Comment:
Please add context in the returned error so we know which code path it came from.